### PR TITLE
Feature/nxl 7901073043 gif save

### PR DESCRIPTION
**Pull request by elusiveuphoria**
## Monday.com Item

Monday Item ID: 7901073043

## Description

Content and preliminary input fields created for GIF save component. Actual GIF generation still needs to be built; successful saves will output filename to console. Using short test array of images as in lieu of data call.

## How Has This Been Tested?

Tested in storybook. Use `npm run storybook` or view preview.

## Screenshots/GIF (if appropriate):
savegif1
savegif2
savegif3
savegif4

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by Panguino — CHANGES_REQUESTED** [review]
I think we could utilize our button component on this.  I am also wondering if this is an opportunity to create an Input component that we re-use in other places we need it.   If you feel like doing that go for it, I will leave that optional.

**Comment by elusiveuphoria** [comment]
@Panguino Review changes based on feedback. Input field broken out into separate component, Button component updated for use in this scenario. Concern mainly with Button, please verify other uses of the Button are unaffected (eg Academics landing, etc).

**Review by Panguino — CHANGES_REQUESTED** [review]
@elusiveuphoria please create storybook stories for the button and input component

**Comment by elusiveuphoria** [comment]
Acted on feedback, see updates to branch. @Panguino

File: src/components/elements/InputText/InputText.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```
Comment (Panguino) [inline src/components/elements/InputText/InputText.tsx]: I think this component is fine, we can always visit this later my only ask is rename it to just `Input` not `InputText`

**Review by Panguino — CHANGES_REQUESTED** [review]
Just 1 change then ready to merge

**Pull request merged**